modules/page_object_fxa_new_account.py

```diff
@@ -0,0 +1,37 @@
+from modules.page_base import BasePage
+
+
+class FxaNewAccount(BasePage):
+    """Page Object Model for FxA signup flow"""
+
+    URL_TEMPLATE = "{fxa_url}"
+
+    def sign_up_sign_in(self, email: str) -> BasePage:
+        """From the entry point, enter email to sign up or sign in"""
+        self.fill("login-email-input", email, press_enter=False)
+        self.get_element("submit-button").click()
+        return self
+
+    def create_new_account(self, password: str, age=30) -> BasePage:
```

Comment:
prehistoric default age 💯  (im just kidding)

Reply:
I almost put my age, which would be even scarier!